css-contain-3/Overview.bs

```diff
@@ -460,6 +461,67 @@ Container Queries: the ''@container'' rule</h2>
 	so it is not always possible to merge the individual <<container-query-list>>s
 	into a single query.
 
+<h3 id="animated-containers">
+Animated Containers</h3>
+
+	A change in the evaluation of a [=container query=] must be treated as a [=style change event=],
```

Comment:
I think "treated as a" should be "part of a", since there's no requirement that the style change event contain *only* that.

But really, the interesting bit is over in #6688, without which you'd also need to define here that the change in the evaluation of the container query is *not* reflected in the updating of the before-change style to the current time.

Reply:
Done.